src/components/MatchCriterion/view/ExistingRecordSections/ExistingSectionFolio.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import {
+  Row,
+  Col,
+  KeyValue, NoValue,
+} from '@folio/stripes/components';
+
+import {
+  Section,
+  FOLIO_RECORD_TYPES,
+} from '../../..';
+
+import { getFieldMatched } from '../../../../utils';
+
+import css from '../../CommonMatchCriterion.css';
+
+export const ExistingSectionFolio = ({
+  existingRecordField,
+  existingRecordType,
+  existingRecordFieldLabel,
+}) => {
+  return (
+    <Section
+      label={existingRecordFieldLabel}
+      className={classnames(css.field, css.inputContainer)}
+    >
+      <Row>
+        <Col xs={12}>
+          <KeyValue value={getFieldMatched(existingRecordField, existingRecordType) || <NoValue />} />
```

Comment:
Do you really need `<KeyValue>` here? You haven't label here, so may be just `getFieldMatched(existingRecordField, existingRecordType) || <NoValue />` would be enouph